### PR TITLE
Update to Zivid SDK v1.8.1

### PR DESCRIPTION
**Pull request by torbsorb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ There are two main categories of samples: **Camera** and **Applications**. The s
 
 - **Camera**
   - **Basic**
-    - **Capture** ([**hdev**][hdev-capture-url]) - This example shows how to acquire a 3D color point cloud from the camera and use it to generate a HALCON ObjectModel3D which is then visualized.
+    - **Capture** ([**hdev**][hdev-capture-url], [**cpp-Zivid**][cpp-captureViaZivid-url], [**cpp-GenICam**][cpp-captureViaGenICam-url]) - This example shows how to acquire a 3D color point cloud from the camera and use it to generate a HALCON ObjectModel3D which is then visualized.
```
Comment (SatjaSivcev) [inline README.md:11]: Does this commit belong here? Are you missing a rebase?

Comment (torbsorb) [reply README.md:11]: that's a local proposal.. was discussing with Christian. Shouldn't have been there

Comment (SatjaSivcev) [reply README.md:11]: So remove or make a different commit?

Comment (torbsorb) [reply README.md:11]: I'll fixup

**Pull request merged**